spot_wrapper/wrapper.py

```diff
@@ -939,9 +1923,11 @@ def _upload_graph_and_snapshots(self, upload_filepath):
         if not localization_state.localization.waypoint_id:
             # The robot is not localized to the newly uploaded graph.
             self._logger.info(
-                "Upload complete! The robot is currently not localized to the map; please localize", \
-                "the robot using commands (2) or (3) before attempting a navigation command.")
+                "Upload complete! The robot is currently not localized to the map; please localize",
+                "the robot using commands (2) or (3) before attempting a navigation command.",
+            )
 
+    @try_claim
```

Comment:
Should this have `try_claim` as a private function?

Reply:
Can you clarify what you mean by having it as a private function? Do you mean an inner function in `_navigate_to`? Is the reason it is necessary for the case where toggling power fails?

Reply:
I meant that since `_navigate_to` was a private function should the supposition be that `try_claim` was already called.